products/sle/main.pm

```diff
@@ -21,6 +21,7 @@ use version_utils
 use File::Find;
 use File::Basename;
 use LWP::Simple 'head';
+use YAML::Tiny;
```

Comment:
Have you evaluated different variants of the YAML libraries? I used `YAML::Syck` because that seemed to be the best compromise. If YAML::Tiny is *enough* though then it's good as well.

Reply:
I've checked two and preferred Tiny as it is the smallest one and the only method we need it LoadFile. If we will come up with a decision to allow import of other files, that call should be just enough. However some libraries (python in this case) support this out of the box.